src/sql/plot.py

```diff
@@ -187,9 +187,9 @@ def _compute_conf_interval(N, med, iqr):
 
     return bxpstats
 
-
 # https://github.com/matplotlib/matplotlib/blob/ddc260ce5a53958839c244c0ef0565160aeec174/lib/matplotlib/axes/_axes.py#L3915
 @requires(["matplotlib"])
+@telemetry.log_call('boxplot')
```

Comment:
Please add tests to make sure the new calls we're making is being called.
You should mock it.

Reply:
I think I can utilize [the testing code](https://github.com/ploomber/core/blob/main/tests/telemetry/test_telemetry.py#L488) from ploomber_core and mock it?

Reply:
Yes, so it should be similar in essence.
This fixture is to make sure the class is mocked (so in other tests there are no telemetry calls).
What essentially we want to achieve with those new tests is mocking and checking if it's called on the functions we expect it to be called.